lib/win32/certstore/mixin/crypto.rb

```diff
@@ -66,6 +66,11 @@ def safe_attach_function(win32_func, *args)
       CERT_NAME_URL_TYPE                                  = 7
       CERT_NAME_UPN_TYPE                                  = 8
 
+      # Retrieve Certificates flag
+      # https://sourceforge.net/p/mingw/mingw-org-wsl/ci/987711554626e4710ad73ce7e44aece511672020/tree/w32api/include/wincrypt.h?format=raw.
```

Comment:
The link in given in the comment downloads `wincrypt.h` file. We should give reference of the Microsoft's official documentation.